### PR TITLE
Added test-build-config.ps1 powershell script to address issue #1002

### PR DESCRIPTION
**Pull request by dragoshsch**
- This PR addresses GitHub issue #1002 by porting [test-build-config]() to a Powershell script

- Jenkinsfile changes will be added in a new PR once [PR #1057](https://github.com/Microsoft/openenclave/pull/1085) is merged.

### CODE REVIEW

**Comment by johnkord** [comment]
The script looks great to me! You will also want to change the Jenkinsfile so it calls this instead of the previous one line command. Do Jenkinsfiles support invoking powershell? Maybe you'll need to invoke powershell first, I'm not sure :)

**Comment by dragoshsch** [comment]
bors try

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -102,9 +102,9 @@ def windowsDebugCrossPlatform() {
     }
     stage('Windows Debug') {
         node('SGXFLC-Windows') {
-           unstash 'linuxdebug'
-           bat 'move build linuxbin'
-           bat 'mkdir build && cd build && cmake -G "Visual Studio 15 2017 Win64" -DADD_WINDOWS_ENCLAVE_TESTS=1 -DLINUX_BIN_DIR=%cd%\\linuxbin\\tests .. && pushd . && "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\Common7\\Tools\\LaunchDevCmd.bat" && popd && cmake --build . --config Debug && ctest -V -C Debug'
+            unstash 'linuxdebug'
+            powershell 'mv build linuxbin'
+            powershell './scripts/test-build-config.ps1 -add_windows_enclave_tests -linux_bin_dir $ENV:WORKSPACE/linuxbin/tests -build_type Debug'
```
Comment (johnkord) [inline .jenkins/Jenkinsfile]: Nice!

**Pull request merged**